### DISCUSSION TITLE
Bústarinn kominn í bæinn - 2023/9/5

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#63] [vefsida-ummæli]
# Bústarinn kominn í bæinn - 2023/9/5

Bústaðu skorið þitt á snapptjatt

https://www.biskupsreglan.com/frettir/B%C3%BAstarinn-kominn-%C3%AD-b%C3%A6inn

**Comment by TJ478** [comment]
Vá, án þín væri ég ekki með 3,5 milljón snapscore. Takk fyrir (það er alls ekki eins og ég fékk það allt áður en þessi atburður gerðist)
